recipes-core/ovmf/ovmf_git.bbappend

```diff
@@ -0,0 +1,78 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/ovmf:"
+
+SRC_URI = "git://github.com/tianocore/edk2.git;branch=master \
+    file://0002-ovmf-update-path-to-native-BaseTools.patch \
+    file://0003-BaseTools-makefile-adjust-to-build-in-under-bitbake.patch \
+    "
```

Comment:
Are the other patches from oe-core no longer needed with the SRCREV specified below?  Usually, a .bbappend just modifies SRC_URI to add or remove patches instead of redefining the entire variable.

This is fine if you want to remove them - it's just a little unconventional.

Reply:
The patches are not relevant to x86_64 builds and they don't apply cleanly to the revision we picked.